mongodb_store/launch/mongodb_store_inc.launch

```diff
@@ -3,6 +3,7 @@
   <!-- This launch file assumes that the machine tag named with the "machine" arg was set previously. If this is not the case then call mongodb_store.launch instead which will set the machine tag for you. This should typically be called within a larger roslaunch structure. -->
 
   <arg name="db_path" default="/var/local/mongodb_store"/>
+  <arg name="host" default="" />
```

Comment:
What if we set it here as:
`<arg name="host" default="$(optenv ROS_HOSTNAME localhost)" />`
And then reference it as:
<param name="mongodb_host" value="$(arg host)" />

That way, the default is `localhost`, which can be overrided by the `ROS_HOSTNAME` environment variable, which can be overrided manually through the argument.

Reply:
Thanks.
I change to use `optenv` for the default value of `mongodb_host` argument.